lib/src/iterable_extensions.dart

```diff
@@ -59,7 +70,21 @@ extension NestedIterableExtensions<E> on Iterable<Iterable<E>> {
   /// elements in iteration order.
   ///
   /// `since 0.0.1`
-  Iterable<E?> get flatten {
```

Comment:
Removing the `?` is considered breaking

Reply:
It's technically a bug since the type parameter on the class is `<E>` not `<E?>`

Reply:
Regardless. Anything that changes the signature or return type is a breaking change. Going from nullable (even if not) to nullable, is a breaking change.

Use of `[1, 2, [3, 4], 5].flatten.first!` (which could have to be done) will throw an exception of null check used on non nullable.

Reply:
Demonstrated this is not a breaking change